### PR TITLE
Update Public API docs with Python API

### PR DESCRIPTION
**Pull request by fabioluz**

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -266,6 +267,40 @@ Deletes a package domain.
 |---------|------------------------------|----------|
 | name    | The full name of the domain  | No       |
 
+### Python Packages
+
+#### List package domains
+`GET https://scarf.sh/api/v1/packages/{package-id}/domains`
+
+List all package domains for the user.
```
Comment (aviaviavi) [inline docs/api.md:275]: @fabioluz do we define what a `package domain` is anywhere? If not, we should probably make sure it's explicitly defined and link to that definition perhaps.

Comment (fabioluz) [reply docs/api.md:275]: We do here, in the same way we do for other package types
image

Comment (fabioluz) [reply docs/api.md:275]: Note that is in the Gateway Docs not in the API Docs

Comment (aviaviavi) [reply docs/api.md:275]: Ok, let's just link to that section here.

**Review by aviaviavi — APPROVED** [review]
Once we add a link on the "domain" reference, this LGTM!

**Pull request merged**